ERCS/erc-7764.md

```diff
@@ -0,0 +1,136 @@
+---
+eip: 7764
+title: Buyer-Seller Negotiable Pricing
+description: Allows buyers and sellers to freely negotiate and determine prices.
```

Comment:
You description doesn't add any new information that wasn't already introduced in your title. Try to elaborate on your title, not just restating it in more words.